### PR TITLE
Use constants for tab-index

### PR DESCRIPTION
**Pull request by centic9**
This makes it easier to change the list of tabs in the future and avoids errors with mixing up indices.

Also fix typo in "addresses"

### CODE REVIEW

File: app/src/main/java/net/programmierecke/radiodroid2/FragmentTabs.java [hunk 1]
```diff
@@ -114,57 +126,58 @@ private void setupViewPager(ViewPager viewPager) {
...
+        fragments[IDX_CHANGED_LATELY] = new FragmentStations();
+        fragments[IDX_CURRENTLY_HEARD] = new FragmentStations();
+        fragments[IDX_TAGS] = new FragmentCategories();
+        fragments[IDX_COUNTRIES] = new FragmentCategories();
+        fragments[IDX_LANGUAGES] = new FragmentCategories();
+        fragments[IDX_SEARCH] = new FragmentStations();
+
+        for (int i=0;i<fragments.length;i++) {
+            Bundle bundle1 = new Bundle();
+            bundle1.putString("url", addresses[i]);
```
Comment (morckx) [inline app/src/main/java/net/programmierecke/radiodroid2/FragmentTabs.java:149]: something like the following seems to be missing here to make search work:
```
if (i == IDX_SEARCH) {
    bundle1.putBoolean(FragmentStations.KEY_SEARCH_ENABLED, true);
}
```

File: app/src/main/java/net/programmierecke/radiodroid2/FragmentTabs.java [hunk 2]
```diff
@@ -114,57 +126,58 @@ private void setupViewPager(ViewPager viewPager) {
...
+        fragments[IDX_TOP_VOTE] = new FragmentStations();
+        fragments[IDX_CHANGED_LATELY] = new FragmentStations();
+        fragments[IDX_CURRENTLY_HEARD] = new FragmentStations();
+        fragments[IDX_TAGS] = new FragmentCategories();
+        fragments[IDX_COUNTRIES] = new FragmentCategories();
+        fragments[IDX_LANGUAGES] = new FragmentCategories();
+        fragments[IDX_SEARCH] = new FragmentStations();
+
+        for (int i=0;i<fragments.length;i++) {
+            Bundle bundle1 = new Bundle();
```
Comment (morckx) [inline app/src/main/java/net/programmierecke/radiodroid2/FragmentTabs.java:148]: why `bundle1`?

**Review by morckx — CHANGES_REQUESTED** [review]
Looks good otherwise and indeed makes the code easier to understand :+1:

File: app/src/main/java/net/programmierecke/radiodroid2/FragmentTabs.java (hunk 1 above)
Comment (centic9) [reply app/src/main/java/net/programmierecke/radiodroid2/FragmentTabs.java]: good catch, applied now.

File: app/src/main/java/net/programmierecke/radiodroid2/FragmentTabs.java (hunk 2 above)
Comment (centic9) [reply app/src/main/java/net/programmierecke/radiodroid2/FragmentTabs.java]: just an oversight, changed now

**Comment by morckx** [comment]
Looks good to me, thanks!

**Comment by centic9** [comment]
Ping?

**Pull request merged**